src/main/java/http/HttpQueryParams.java

```diff
@@ -0,0 +1,44 @@
+package http;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpQueryParams {
+    private static final String QUERY_STRING_DELIMITER = "&";
+    private static final String KEY_VALUE_DELIMITER = "=";
+    private static final String QUERY_DELIMITER = "\\?";
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
+    private final Map<String, String> queryParam;
+
+    public HttpQueryParams() {
+        this.queryParam = new HashMap<>();
+    }
+
+    public static HttpQueryParams of(String path) {
+        String[] paths;
+        paths = path.split(QUERY_DELIMITER);
```

Comment:
가급적 변수를 선언과 동시에 할당도록 해보시면 좋을 것 같아요. 🙂
```suggestion
        String[] paths = path.split(QUERY_DELIMITER);
```